### PR TITLE
Update next branch docs with links to resources about v10

### PR DESCRIPTION
**Pull request by colelawrence**
## 🎯 Changes

Make it easier to learn about v10.

## ✅ Checklist

- [x] I have followed the steps listed in the [Contributing guide](https://github.com/trpc/trpc/blob/main/CONTRIBUTING.md).
- [x] If necessary, I have added documentation related to the changes made.
- [x] I have added or updated the tests related to the changes made.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+> **🚀 You are looking at a pre-prelease of tRPC v10!**
```
Comment (sachinraja) [inline README.md:1]: Is `pre-prelease` intentional wording? I know this was copied from the header but we should probably fix it.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+> See our [v10-docs.md](./.tmp/v10-docs.md) for a summary of what is changing or take a look at [the **v10 docs website**](https://alpha.trpc.io/).
```
Comment (sachinraja) [inline README.md:2]: I'm not sure if those docs are a good source anymore vs the migration guide on the website

File: README.md (hunk 1 above)
Comment (KATT) [reply README.md:1]: It's okay

File: README.md (hunk 2 above)
Comment (KATT) [reply README.md:2]: Yeah let's just link the migration guide. I think we can remove that doc now?

**Comment by colelawrence** [comment]
Thanks @sachinraja and @KATT, I think this will be really helpful for others interested in providing feedback for future trpc versions.

**Pull request merged**